crates/matrix-sdk-ui/tests/integration/timeline/replies.rs

```diff
@@ -350,7 +350,13 @@ async fn test_fetch_details_utd() {
         let in_reply_to = in_reply_to.clone().unwrap();
         assert_let!(TimelineDetails::Ready(replied_to) = &in_reply_to.event);
         assert_eq!(replied_to.sender(), *ALICE);
-        assert!(replied_to.content().is_unable_to_decrypt());
```

Comment:
Why change this? 🤔

Reply:
Better error, when the things don't match (spoiler: it didn't match, at some point, during dev). It's easy to revert, though, if you feel strongly about it.

Reply:
No no, just curious